hkm/models/models.py

```diff
@@ -62,7 +62,7 @@ class UserProfile(BaseModel):
     language = models.CharField(verbose_name=_(u'Language'), default=LANG_FI,
                                 choices=LANGUAGE_CHOICES, max_length=4)
     is_museum = models.BooleanField(verbose_name=_(u'Is museum'), default=False)
-    printer_ip = models.IPAddressField(verbose_name=u"Museum printer IP", blank=True, null=True)
```

Comment:
Had to check, `IPAddressField` was removed in Django 1.9 (only supported for historical migrations). So this was necessary change. :+1: